compliance_checker/imos/imos.py

```diff
@@ -1255,8 +1255,8 @@ def check_data_variables(self, dataset):
 
     def check_quality_control_conventions_for_quality_control_variable(self, dataset):
         """
-        Check that the attributes quality_control_set and quality_control_conventions
-        are valid and consistent.
+        Check that the attribute quality_control_conventions
+        is valid and consistent.
         """
         test_value_dict = {'1': "IMOS standard set using the IODE flags",\
```

Comment:
This doesn't need to be a dict now. A tuple would be ideal (e.g. `qc_conventions = ("IMOS standard set using the IODE flags", etc...)`